### ISSUE TITLE
Organize import at package/folder level

### ISSUE THREAD

**Issue by Osolemio44** [redhat-developer/vscode-java#2282]
It is very annoying when you need to do an "organize imports" in several files and the only option is to go file by file. This option should exist at the folder level to be able to apply it at once to all the files inside. Or to be able to select several files and allow you to apply it to them. I have seen that there is another post ( #1107 )  from 2019 that talks about this topic, but has no answer.

**Comment by rgrunber** [comment]
Right. Looks like this is a feature that is requested enough that we should have a look at it.

I'll mark this particular issue as a duplicate of #1107, and we can track progress there.
